libraries/Microsoft.Bot.Schema/Teams/MeetingParticipantsJoinEventDetails.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
```

Comment:
System and Newtonsoft are not necessary.

Reply:
Newtonsoft is being used for the JsonProperty tag

Reply:
Which tag?
![image](https://github.com/southworks/botbuilder-dotnet/assets/44245136/f939171d-39fd-4589-988f-c32f365e0819)